cmake/Modules/CopyMSVCBins.cmake

```diff
@@ -89,13 +89,9 @@ file(
   "${FFMPEG_avcodec_INCLUDE_DIR}/bin${_bin_suffix}/libbz2*.dll"
   "${FFMPEG_avcodec_INCLUDE_DIR}/bin${_bin_suffix}/zlib*.dll")
 
-file(
-  GLOB
-  X264_BIN_FILES
-  "${X264_INCLUDE_DIR}/../bin${_bin_suffix}/libx264-*.dll"
-  "${X264_INCLUDE_DIR}/../bin/libx264-*.dll"
-  "${X264_INCLUDE_DIR}/bin/libx264-*.dll"
-  "${X264_INCLUDE_DIR}/bin${_bin_suffix}/libx264-*.dll")
+file(GLOB X264_BIN_FILES "${X264_INCLUDE_DIR}/../bin${_bin_suffix}/libx264-*.dll"
+     "${X264_INCLUDE_DIR}/../bin/libx264-*.dll" "${X264_INCLUDE_DIR}/bin/libx264-*.dll"
+     "${X264_INCLUDE_DIR}/bin${_bin_suffix}/libx264-*.dll")
```

Comment:
Do the formatting tools really squash this?

Reply:
Yes, because command formatting also takes the amount of tokens into accounts - libx264 doesn't have enough tokens to warrant the "per-line" form with its default settings.

Reply:
Ah, got it.